content/en/contribute/_index.md

```diff
@@ -7,36 +7,105 @@ menu:
     pre: <i class='fas fa-pen-square'></i>
 ---
 
-All the content for the Cloud Native Glossary is stored in [this GitHub repo](https://github.com/cncf/glossary).  You'll find there a list of [issues](https://github.com/cncf/glossary/issues), [PRs](https://github.com/cncf/glossary/pulls), and [discussions](https://github.com/cncf/glossary/discussions) about the glossary.
+The Cloud Native Glossary content is stored in [this GitHub repo](https://github.com/cncf/glossary) where you'll find a list of [issues](https://github.com/cncf/glossary/issues), [PRs](https://github.com/cncf/glossary/pulls), and [discussions](https://github.com/cncf/glossary/discussions) about the glossary. 
 
-## General guidelines
-To propose specific changes to a glossary entry, edit that entry in your branch and issue a pull request. To request that an entry be clarified, updated, or reconsidered, you may alternatively open an issue. To propose a new entry to the glossary, either create an issue or, if you have the definition drafted, add the entry to your branch and create a pull request.
+There are three ways you can contribute:
 
+1) Propose new terms
+2) Update existing ones 
+3) Help translate the glossary
 
-## Issues
+## Join the Glossary community! 
+Consider joining our monthly Glossary Working Group meetings if you want to contribute regularly. You can find meeting details in the CNCF calendar. You can also connect with the maintainers and fellow contributors in our #glossary channel on the [CNCF calendar](https://www.cncf.io/calendar/) — we'd love to meet you! 
 
-Please jump in and help us out by reviewing [open issues](https://github.com/cncf/glossary/issues) and making pull requests to resolve them.  The easiest ones have been marked with the “good first issue” or “help wanted” tags.  Choose one that hasn't already been assigned to someone:
+## Propose new terms
+You can propose a new term for others to work on or create a new definition yourself. Either way, you'll start by creating an issue (note that you'll need a GitHub account to do this).
 
-![issues](/images/how-to/3.png)
+Below is a step-by-step guide for those not yet familiar with GitHub. **If you are a GitHub Pro**, please *do* have a quick look to make sure you use our issue templates, appropriate naming conventions, claim a PR on Slack (otherwise we may miss it), and where to find the file template. And please make sure to read the [Style Guide](https://glossary.cncf.io/style-guide/) before getting started — thank you! 
 
-Other that haven't yet been assigned may have been "claimed" by someone. Click on the issue to learn more about it. The example below is already claimed:
+### Creating an issue
+Go to the [Glossary GitHub repo](https://github.com/cncf/glossary/issues) issues and click on "new issue."
 
-![claims](/images/how-to/4.png)
+ADD IMAGE 1
```

Comment:
@CathPag PTAL #325 to add image files for how-go guide.

After PR #325 is merged, 

You need to update `ADD IMAGE xx` parts like this ..

```suggestion
![howto-01](/images/how-to/howto-01.png)
```